plugins/coral-plugin-offtopic/client/components/OffTopicFilter.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styles from './styles.css';
+
+export default class OffTopicFilter extends React.Component {
+
+  tag = 'OFF_TOPIC';
+  className = 'offTopicComment';
```

Comment:
Yes, just need a sync with riley + david for the current naming scheme. They've been working on that.